docs/_threats/tr-1.md

```diff
@@ -9,4 +9,16 @@ external-refs:
     - "LLM06: Sensitive Information Disclosure"
 ---
 
-Proprietary information from within the client organisation must be processed externally by chat.com. Insufficient controls with and within chat.com result in a sensitive information disclosure risk.
+In the provided system architecture, sensitive data is transmitted to a SaaS-based Generative AI platform (e.g., chat.com) for inference, posing a risk of information leakage. Sensitive organizational data, proprietary algorithms, and confidential information may be unintentionally exposed due to inadequate control measures within the hosted model. This can occur through several mechanisms unique to Large Language Models (LLMs), as outlined in OWASP's LLM06, such as [overfitting](https://aws.amazon.com/what-is/overfitting/), [memorization](https://arxiv.org/pdf/2310.18362), and [prompt-based attacks](https://owasp.org/www-project-llm-prompt-hacking/).
+
+LLMs can retain data from training processes or user interactions, potentially recalling sensitive information during unrelated sessions, a phenomenon known as "memorization." When data such as Personally Identifiable Information (PII) or proprietary financial strategies enter the model, the risk of inadvertent disclosure rises, particularly when insufficient data sanitization or filtering mechanisms are in place. Additionally, adversarial actors could exploit prompt injection attacks to manipulate the model into revealing sensitive data.
```

Comment:
Regarding memorization, this is an interesting point, and one where I think there is some confusion. There is a common misconception that all hosted LLMs are training on all the input data (prompts) sent to them. This is not the case, for some (e.g. ChatGPT) you can opt-out, for enterprise / commercial products they don't use your data for training.

I think we need to draw a clear distinction here between the risks associated with either free hosted LLMs, and the more enterprise products.

And of course, highlight teh risk of not reading the terms!

Reply:
Just added an additional paragraph that better explains this consideration. Let me know if you have any further suggestions!